packages/orbit-components/src/Tile/index.js

```diff
@@ -53,7 +53,7 @@ const Tile = ({
       onKeyDown={handleKeyDown(onClick)}
       as={href ? "a" : "div"}
       tabIndex={!href ? "0" : undefined}
-      role={!href ? "button" : undefined}
+      role={href ? "link" : "button"}
```

Comment:
`TileWrapper` will be `<a>` if it has `href`, so setting `role` to `link` is redundant and should remain `undefined`, tests pass regardless of this change.